frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -942,11 +980,10 @@ pub mod pallet {
 			// Note: we don't `rotate_round` at this point; the next call to
 			// `ElectionProvider::elect` will succeed and take care of that.
 
-			let solution = ReadySolution {
-				supports,
-				score: Default::default(),
-				compute: ElectionCompute::Emergency,
-			};
+			let solution: ReadySolution<_, _> =
+				(supports, Default::default(), ElectionCompute::Emergency)
+					.try_into()
+					.map_err(|_| Error::<T>::TooManyBackersPerWinner)?;
```

Comment:
```suggestion
ReadySolution {
  supports: supports.into_iter().map(|s| s.try_into()).collect::<Result<Vec<_>, _>>(),
  score: Default::default(),
  ElectionCompute::Emergency,
};
```

Supplement of https://github.com/paritytech/substrate/pull/11935/files#r933816968